map/place_page_info.cpp

```diff
@@ -97,5 +99,34 @@ string Info::GetCustomName() const { return m_customName; }
 BookmarkAndCategory Info::GetBookmarkAndCategory() const { return m_bac; }
 string Info::GetBookmarkCategoryName() const { return m_bookmarkCategoryName; }
 string const & Info::GetApiUrl() const { return m_apiUrl; }
+
+string Info::GetRatingFormatted() const
+{
+  if (!IsBookingObject())
+    return string();
+
+  auto const r = GetMetadata().Get(feature::Metadata::FMD_RATING);
+  char const * rating = r.empty() ? kEmptyRatingSymbol : r.c_str();
+  int const sz = snprintf(nullptr, 0, m_localizedRatingString.c_str(), rating);
+
+  string buf(sz + 1, ' ');
+  snprintf(&buf[0], buf.size(), m_localizedRatingString.c_str(), rating);
+  return buf;
+}
+
+string Info::GetApproximatelyPricing() const
+{
+  if (!IsBookingObject())
+    return string();
+
+  uint64_t pricing;
+  strings::to_uint64(GetMetadata().Get(feature::Metadata::FMD_PRICE_RATE), pricing);
```

Comment:
Do you really need uint64 here? Or int would be enough? Do we have some upper bound on pricing? Why no to add an assertion here?